frontend/src/pages/CustomImages/ImageBuildDialog.js

```diff
@@ -148,10 +149,9 @@ export default function ImageBuildDialog(props) {
             <Input value={imageId} onChange={({ detail }) => {setImageId(detail.value)}} />
           </FormField>
         </div>
-        <textarea
-          spellCheck="false"
-          className="configuration-data" value={imageConfig}
-          onChange={(e) => {setState([...imageBuildPath, 'config'], e.target.value)}} />
+        {<ConfigView
+          config={imageConfig}
+          onChange={(e) => {setState([...imageBuildPath, 'config'], e.target.value)}}/>}
```

Comment:
I think the `setState` needs to change to `e.detail.value` per this: https://github.com/aws-samples/pcluster-manager/blob/main/frontend/src/pages/Configure/Create.js#L103

it's a little weird because the editor "works" but it isn't _actually_ changing the underlying value.

Reply:
`onChange={({detail}) => {setState([...imageBuildPath, 'config'], detail.value)}}/>}`

Reply:
All up to date!